### ISSUE TITLE
Support for RHEL ( RedHat Enterprise Linux )

### ISSUE THREAD

**Issue by linuxy9** [pterodactyl/panel#1062]
when you try to install rhel. you get a unsupported os on the daemon. why? rhel 7.4 is almost the same as cento 7.4

why can't we  have both

what I oone so far that with the panel

- panel works ( epel and extra repos required with ius to install php72u)
- docker-ce ( with the rhel repos to install selinux-container )
- node.js works

the only thing does not work is the daemon

thanks in the future

**Comment by parkervcp** [comment]
@linuxy9 Can you give us the complete error please? Just giving us a "not supported" doesn't help.

**Comment by lancepioch** [comment]
Feel free to reopen this with more details

**Comment by linuxy9** [comment]
Reopening the ticket:
after what dane said:
![snapshot1](https://user-images.githubusercontent.com/2430792/37576225-0a9afcd0-2af1-11e8-9fc8-19d124d814aa.png)


I get the error of:
`Error: Unable to create a pterodactyl user and group, unknown operating system.`

On RedHat Enterprise Linux 7.4 Server

**Comment by linuxy9** [comment]
And RHEL worked on your panel before. Idk what changed

**Comment by schrej** [comment]
You should be able to reopen the issue yourself. You also should've added more information as requested.

**Comment by DoNotSpamPls** [comment]
This seems to just be an issue of the daemon not knowing about RHEL, even though it's probably supported (since CentOS works)

This is the code in question

```
switch (_.get(os, 'dist')) {
case 'Ubuntu Linux':
case 'Debian':
case 'Centos':
       UserCommand = `useradd --system --no-create-home --shell /bin/false ${Username}`;
       break;
default:
       return scall(new Error('Unable to create a pterodactyl user and group, unknown operating system.'));
}
```

I'm pretty sure RHEL just needs to be added to that list, and the command is probably the same as CentOS, since it's based off RHEL. I can't test this though, so probably someone smarter than me can fix it.

**Comment by parkervcp** [comment]
@DoNotSpamPls I already got that far. Was waitig on Linuxy to test a fix.. They just rolled centos instead.

**Comment by linuxy9** [comment]
- REOPENING THE ISSUE -

Due to the fact that this issue is not done yet of testing on RHEL 7.4/7.5 ( RedHat Enterprise Linux ). And **I got super busy in life that I could not do the Issue. And I want to Apologies for that!**


I am reopening the issue. 


My plan for testing is to run a VM on VMware ESXi with RHEL 7.5 ( RedHat Enterprise Linux ) with the Panel and Daemon!

Then post a tutorial for you guys to look over. What I did and how to do it.


Any issues with the setup will be posted here!
